tools/Microsoft.Health.Dicom.SchemaManager/OptionAliases.cs

```diff
@@ -0,0 +1,24 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Dicom.SchemaManager;
+
+public static class OptionAliases
+{
+    public const string Version = "--version";
+    public const string ShortVersion = "-v";
```

Comment:
nit: can we rename the short versions to have a `Short` suffix instead of prefix and then alphabetize this list?